### ISSUE TITLE
Add <-- Back button on Settings page

### ISSUE THREAD

**Issue by AndersonCeci** [OpenCovenant/quill#246]

**Comment by vaqueraoscar0** [comment]
@AndersonCeci Maybe the setting's button can serve as both, taking the user to the settings page, as well as taking him back to the homepage. That might give it a cleaner look.

**Comment by AndiBraimllari** [comment]
Interesting idea, I would be fine with this as an intermediate solution.

Btw @vaqueraoscar0, we're open to any ideas in regard to how we can properly convey to the user the message that they've hit the limit of 10k characters in the editor? We're looking for something minimalistic and clean, yet visible (maybe a simple span of text of different color shown somewhere?). Maybe you'll have a good idea here as well.

**Comment by vaqueraoscar0** [comment]
@AndiBraimllari Yeah i see the issue there, how about adding a border color of red to the box after they've exceed the character limit and temporarily replace the character count that’s located at the bottom with the error message, you can maybe have it there for as long as the character count is exceeded and have it fade out after they remove some characters. the span text is a good idea as well.

So something along the lines of this, 
<img width="1191" alt="Screenshot 2023-09-27 at 11 50 41 AM" src="https://github.com/OpenCovenant/quill/assets/57508905/be65140c-ccfd-4eb4-8276-8ec993b5fe29">

**Comment by AndersonCeci** [comment]
> @AndiBraimllari Yeah i see the issue there, how about adding a border color of red to the box after they've exceed the character limit and temporarily replace the character count that’s located at the bottom with the error message, you can maybe have it there for as long as the character count is exceeded and have it fade out after they remove some characters. the span text is a good idea as well.
> 
> So something along the lines of this, <img alt="Screenshot 2023-09-27 at 11 50 41 AM" width="1191" src="https://user-images.githubusercontent.com/57508905/271087059-be65140c-ccfd-4eb4-8276-8ec993b5fe29.png">

I appreciate your input. I am currently considering simply coloring the error message in red should be adequate without the need for a border. To further alert the user when they reach the limit, a gentle shaking effect perhaps can be implemented to grab their attention. I'll try this out and see how it looks.

**Comment by AndersonCeci** [comment]
> @AndiBraimllari Yeah i see the issue there, how about adding a border color of red to the box after they've exceed the character limit and temporarily replace the character count that’s located at the bottom with the error message, you can maybe have it there for as long as the character count is exceeded and have it fade out after they remove some characters. the span text is a good idea as well.
> 
> So something along the lines of this, <img alt="Screenshot 2023-09-27 at 11 50 41 AM" width="1191" src="https://user-images.githubusercontent.com/57508905/271087059-be65140c-ccfd-4eb4-8276-8ec993b5fe29.png">

I appreciate your input. I am currently considering simply coloring the error message in red should be adequate without the need for a border. To further alert the user when they reach the limit, a gentle shaking effect perhaps can be implemented to grab their attention. I'll try this out and see how it looks.
